USFMConverter/Core/Data/Setting.cs

```diff
@@ -0,0 +1,35 @@
+using USFMConverter.UI;
+
+namespace USFMConverter.Core.Data
+{
+    public class Setting
+    {
+        public int TextSizeIndex { get; set; }
+        public int LineSpacingIndex { get; set; }
+        public int ColumnCountIndex { get; set; }
+        public bool Justified { get; set; }
+        public bool LeftToRight { get; set; }
+        public bool ChapterBreak { get; set; }
+        public bool VerseBreak { get; set; }
+        public bool NoteTaking { get; set; }
+        public bool TableOfContents { get; set; }
+
+        public Setting()
+        {
+            
+        }
+        public Setting(ViewData? dataContext)
```

Comment:
I would not recommend having single-nullable parameter like this in the constructor. It turns out that when you or someone initializes this object with a null argument, it will be the same as the default empty constructor above... 

Please remove the nullable "?" type here